src/transformers/file_utils.py

```diff
@@ -229,7 +229,12 @@
 
 S3_BUCKET_PREFIX = "https://s3.amazonaws.com/models.huggingface.co/bert"
 CLOUDFRONT_DISTRIB_PREFIX = "https://cdn.huggingface.co"
-HUGGINGFACE_CO_PREFIX = "https://huggingface.co/{model_id}/resolve/{revision}/{filename}"
+
+_staging_mode = os.environ.get("HUGGINGFACE_CO_STAGING", "NO").upper() in ENV_VARS_TRUE_VALUES
+if _staging_mode:
+    HUGGINGFACE_CO_PREFIX = "https://moon-staging.huggingface.co/{model_id}/resolve/{revision}/{filename}"
+else:
+    HUGGINGFACE_CO_PREFIX = "https://huggingface.co/{model_id}/resolve/{revision}/{filename}"
```

Comment:
Nit, but maybe pass the endpoint itself as an env variable?

like
```
HUGGINGFACE_CO_RESOLVE_ENDPOINT = os.environ.get("HUGGINGFACE_CO_RESOLVE_ENDPOINT", "https://huggingface.co")

HUGGINGFACE_CO_PREFIX = HUGGINGFACE_CO_RESOLVE_ENDPOINT + "/{model_id}/resolve/{revision}/{filename}"
```

That way we don't have to hardcode the moon-landing endpoint below ⤵️

Reply:
Will still keep my `os.environ.get("HUGGINGFACE_CO_STAGING", "NO").upper()` as it's what is used in the tests and use the result to set default of `HUGGINGFACE_CO_RESOLVE_ENDPOINT` if no env variable is set, does that sound ok?